### ISSUE TITLE
YouTube Field Colorbox activation errror

### ISSUE THREAD

**Issue by amilenkov** [backdrop-contrib/youtube#29]
Hello,

I'm trying to enable the YouTube Field Colorbox module and I'm getting the following error message:

Error
syntax error, unexpected '$domain' (T_VARIABLE)

Returning to the modules page, the module is not active.

I checked the documentation and read the README.md file in a folder and followed the directions:

Usage
------
To use this module:
  1. The Colorbox plugin must be properly included. The status of this can be
     checked at admin/reports/status.
  2. 'Enable Colorbox load' must be enabled on the Colorbox module settings. This
     setting is found at admin/config/media/colorbox.

Same error, I can't activate the YouTube Field Colorbox module

I am attaching a screenshot:

![Firefox_Screenshot_2022-11-27T11-07-05 265Z](https://user-images.githubusercontent.com/14857883/204131968-a2d38fec-fe13-4dc1-a566-736864a6d832.png)

**Comment by olafgrabienski** [comment]
Hm, I can't reproduce the issue at first sight. On a vanilla Backdrop demo site I was able to enable the *YouTube Field Colorbox* module without problems. Here my steps:

- Start a demo site on https://backdropcms.org/demo.
- Go to the Project browser, visit the *Install modules* page (`admin/modules/install`), install and enable the (independent) *Colorbox* module and the *YouTube Field* module.
- Enable also the (*YouTube* submodule) *YouTube Field Colorbox* module.

**Comment by amilenkov** [comment]
Hi, thanks for the reply. This led me to carefully review everything related to YouTube Field and its configuration on the site where I noticed the problem. I've been using this module - YouTube Field for many years and now I've found problems that I guess have developed over the years, I just haven't activated YouTube Field Colorbox until now and that's probably why I didn't notice them.

First, a strange version of the module was installed on my site - 1.23.0 - I don't know how it was possible, when I downloaded the module from the official Backdrop modules page, and at the moment the latest version there is 1.x-1.0 .2.

Here is a screenshot of the installed module that was causing the problem:

![Firefox_Screenshot_2022-11-28T09-58-19 774Z](https://user-images.githubusercontent.com/14857883/204253574-83ae3dbf-2f85-4b6c-95e9-782d9d50606e.png)

Anyway, the problem was solved after I uninstalled the module, downloaded version 1.x-1.0.2 and installed it. Now the YouTube Field Colorbox is activated and the error message is gone.

I just don't know where this 1.23.0 version came from that was activated before. It also required installing the Libraries module and the Colorbox library.

The second screenshot shows the version of the module where the problem disappeared.

![Firefox_Screenshot_2022-11-28T10-25-18 333Z](https://user-images.githubusercontent.com/14857883/204255348-43c87049-b208-4c99-bbb4-e80609d20e69.png)
